### ISSUE TITLE
RAM Unit

### ISSUE THREAD

**Issue by ericrange** [ericrange/Melinda#3]
Do we need the ram to be in bytes? or can we just store the ram in MB? (saves storage)

**Comment by PabloSichert** [comment]
Assuming we use MongoDB, integers will be stored as a 64 bit signed integer. Thus, the same memory would be allocated and it is no problem saving the number of bytes as a whole.

Tracking 18,446,744,073,709,551,615 bytes ^= 8192 peta bytes of RAM should be enough for now. :smile: 

http://api.mongodb.org/perl/current/MongoDB/DataTypes.html

**Comment by ericrange** [comment]
but it will reduce the amount of json. there is a difference between
```
{
     ram_used: 518371651938 (vs)
     ram_used: 511647
}
```

**Comment by PabloSichert** [comment]
That's true. I would still say that is an overhead we can take as it just marginally affects the bandwith and not even the storage.
But I'm rather open-minded to this question. Saving the RAM used in MB would be reasonable as we don't claim to track it in such a perfect accuracy.
